components/d2l-applied-filters/d2l-applied-filters.js

```diff
@@ -233,7 +232,7 @@ class D2lAppliedFilters extends RtlMixin(LocalizeStaticMixin(LitElement)) {
 	}
 
 	_setSelectedOptions() {
-		this._selectedEntries = [].concat.apply([], Object.values(this._entries || {})).filter(x => x.selected);
+		this._selectedEntries = [].concat(... Object.values(this._entries || {})).filter(x => x.selected);
```

Comment:
This failed the [`prefer-spread`](https://eslint.org/docs/rules/prefer-spread) rule